### PR TITLE
Several improvements for Daily theme

### PR DESCRIPTION
**Pull request by ljubar**

### CODE REVIEW

File: views/article.html.twig [hunk 1]
```diff
@@ -83,6 +82,17 @@
 						</div>
 						#}
 
+						{% for keyword in gimme.article.keywords %}
```
Comment (ahilles107) [inline views/article.html.twig]: Indentations shoud be fixed.

File: views/partials/mainArticle--small.html.twig [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<article class="mainArticle {{class}}">
+
+            <figure class="mainArticle__image {% if image =='small' %}mainArticle__image--small{% endif %} ">
+
+{% set imgCaption = "" %}
+{% set rend610 = "/public/img/article_img/610x380.png" %}
+{% set rend1250 = "/public/img/article_img/1250x600.png" %}
+
+  {% if article.featureMedia %}
```
Comment (ahilles107) [inline views/partials/mainArticle--small.html.twig:9]: you can move it to macro or separate file (to be included).

File: views/partials/user-profile.html.twig [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+					</div>  
+
+				</form>	
+
+				<div id="editProfileResult" class="notification"></div>			
+
+			</div>
+		</div>
+	</div>
+</div>
```
Comment (ahilles107) [inline views/partials/user-profile.html.twig]: missing new line at end of file

File: views/partials/register-popup.html.twig [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+			</form>
+
+			<div id="registrationResult" class="notification"></div>
+
+		</div>
+
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (ahilles107) [inline views/partials/register-popup.html.twig]: missing new line at end of file

File: views/partials/login-popup.html.twig [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+			</form>
+
+			<div id="userAuth" class="notification"></div>
+
+		</div>
+
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (ahilles107) [inline views/partials/login-popup.html.twig]: missing new line at end of file

**Comment by ljubar** [comment]
@ahilles107 changes committed - pls review

**Pull request merged**